pkg/hoist/hoist_launchable_test.go

```diff
@@ -14,6 +15,8 @@ import (
 	. "github.com/anthonybishopric/gotcha"
 )
 
+var terminationGracePeriod = 1 * time.Hour
```

Comment:
I'm still not thrilled with us defaulting to 1 hour, but I think I lost that battle before I joined 😢